CRFModuleValidation/Cardio_12MT.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+<title>Instructions</title>
+<link href="../Learn_section.css" rel="stylesheet" type="text/css" />
+</head>
+
+<body>
+
+<div class="container">
+    <div class="content">
+        <h1>About 12 minute test</h1>
+        <h2>Why this test</h2>
+        <p>Recording a 12 minute  test is an important measure of your overall health. It is associated with overall mortality and many common diseases in the US such as cardiovascular disease and Alzheimer’s Disease.</p>
+        <h2>What to do</h2>
+        <ol>
+            <li>Measure your <strong>heart rate</strong> within this app before running for 12 minutes.</li>
```

Comment:
since "after" is part of the bolded text in the last list item, should "before" be bolded here?

Reply:
No idea why they picked those words to be strong. It looks kind of weird and I'm guessing will need to be word-smithed. I'm guessing that they are trying to highlight why they take your heart rate twice.